### PR TITLE
Feature Locale of Dates and Commands in en-US and es-ES

### PR DESCRIPTION
**Pull request by javiersigler**
Hi @evancohen,

First of all, I'd like to congratulate you for your excellent job in this project. After a couple of weeks of researching, I truly think this is one of the best solutions to the Smart Mirror project.

You will find in this PR a possible solution to the internationalization problem. All text related to dates are totally translated based on the internationalization of angular and moment. However, there are some texts that are not totally translated like "Time in" or the initial message. I hope to do it in the next PR.

I'mt totally a beginner in Javascript or AngularJS, so any comment is absolutely welcome.

### CODE REVIEW

**Comment by evancohen** [comment]
Awesome work! I already fixed a few of these things in my last PR, looks like there is a merge conflict now. Please fix so I can merge :)

File: js/annyang.js [hunk 1]
```diff
@@ -136,7 +136,7 @@
       recognition.interimResults = true;
 
       // Sets the language to the default 'en-US'. This can be changed with annyang.setLanguage()
-      recognition.lang = 'en-US';
+      recognition.lang = config.locale;
```
Comment (evancohen) [inline js/annyang.js]: This should be set in the Annyang Service. Not in the Annyang library (which may be updated one day).

**Comment by javiersigler** [comment]
Thank you @evancohen, what do u think of this approach?

**Comment by evancohen** [comment]
Totally loving `moment-with-locales.js`. I don't have the time to look at the rest right this second, but I'll try to merge it in today :)

**Comment by ctutu** [comment]
Hello,  very good job!!! If you want I can translate in french ?

**Comment by evancohen** [comment]
You'll need to add a French locale file. Look at the Spanish and English ones for an example of the format.

**Comment by FrankMatheis** [comment]
Great Job! I already have a german locale file - if you want.

**Comment by evancohen** [comment]
@FrankMatheis please send a pull request against the `dev` branch!

**Comment by ctutu** [comment]
I saw is missing in dev branch moment local reference in index.html depending on language config settings.
Actually, no time, week day are translated.

**Comment by ctutu** [comment]
@evancohen how can I send you french translatation file ?

**Pull request merged**